### DISCUSSION TITLE
Save/update runtime configuration in file for unexpected restarts

### DISCUSSION THREAD

**Discussion by JeDiE99** [bluenviron/mediamtx#1050] [Questions]
Save runtime configuration in file after http api requests and read on start by flag ?
This increase service stability

**Comment by aler9** [comment]
Hello, at the moment any configuration change performed with the API is not saved to disk. My advice is to use a custom service in order to store configuration somewhere (for instance, on a DB) and apply it after every server restart. This may change in the future.

**Comment by JeDiE99** [comment]
Thanks for the answer
That's what I thought when I looked at the documentation

[a comment by an automated account is not shown]
